client/components/form.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+
+const Form = () => {
+
+  const [name, setName] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const [barberName, setBarberName] = useState('');
+
+  const submitForm = async event => {
+    event.preventDefault();
+    try {
+      const body = { name, phoneNumber, barberName };
+      // eslint-disable-next-line
```

Comment:
I don't think this eslint directive is necessary.

Reply:
I don't call response in the function so it underlines the variable warning me it isn't used